pkg/services/ngalert/api/tooling/definitions/provisioning_alert_rules.go

```diff
@@ -111,6 +111,8 @@ type ProvisionedAlertRule struct {
 	Labels map[string]string `json:"labels,omitempty"`
 	// readonly: true
 	Provenance models.Provenance `json:"provenance,omitempty"`
+	// example: true
+	IsPaused bool `json:"isPaused"`
 }
 
 func (a *ProvisionedAlertRule) UpstreamModel() (models.AlertRule, error) {
```

Comment:
There should also be a change in this function, right now this makes it so the provisioning API can read whether a rule is paused, but the field will not be mapped (and remain false) if something tries to set the paused flag through the prov API.